app/views/courses/index.html.erb

```diff
@@ -24,7 +24,7 @@
 
             <% if category != :completed %>
               <div class="collection blue darken-3">
-                <% course.current_assessments(DateTime.now, CourseUserDatum.where("user_id = ? AND course_id = ?", @current_user.id, course.id).first).each do |asmt| %>
+                <% course.current_assessments(DateTime.now, course_cud).each do |asmt| %>
```

Comment:
Why did the user and course specification get removed?

Reply:
course_cud was already available so swapping saves an extra DB call in this file